src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -642,17 +642,19 @@ public static double getPlayerVisibilityDistance(EntityPlayer player, double xzD
         return value >= maxXZDistance ? maxXZDistance : value;
     }
 
-    public static boolean isLivingOnLadder(@Nonnull IBlockState state, @Nonnull World world, @Nonnull BlockPos pos, @Nonnull EntityLivingBase entity)
```

Comment:
Removing this breaks binary compatability...
Deprecate and call through to the new one with an null side argument?